lib/Unix/PID/Tiny.pm

```diff
@@ -102,11 +135,59 @@ sub get_pid_from_pidfile {
     return int( abs($pid) );
 }
 
+sub _sets_match {
+    my ( $a, $b ) = @_;
```

Comment:
`$a` and `$b` are special variables, might be good to name them something else?

Reply:
They're not really special. I named the variables following the convention of the Perl sort() function in block form.

Reply:
that’s just it, they are special to `sort()`, it owns them so we get can get odd behavior if a sort is every added. best to avoid them outside of a sort block.

Reply:
Oh! I see what you mean now. Still, this will only impact _sets_match() if it uses sort internally. Perl will complain VERY loudly if you use a block sort in a lexical scope where $a and $b are declared, however. I'm leaving it as-is for the time being.

![Screen Shot 2019-04-23 at 17 05 59](https://user-images.githubusercontent.com/234407/56619096-145e7c80-65ea-11e9-979a-2ae022efc193.png)

Reply:
OK, I'll change it before it goes to CPAN then. I really don't like `$a` and `$b` outside of sort being public.